templates/opendkim/KeyTable

```diff
@@ -1,3 +1,5 @@
+### THIS FILE IS MANAGED BY ANSIBLE -- DO NOT EDIT! ###
```

Comment:
Isn't that already done in our ansible repos? I remember the header looking like `{{ ansible_managed }}`.

Reply:
Or does that only cover `ansible.git`, but not the `ansible-mta` submodule?

Reply:
Usually, I would also vote for `{{ ansible_managed }}`. There are two reason for not using it:

1. It only works in templates and not in static files.
2. All other roles in our ansible repo use this static line. I just copied it. :)